### PR TITLE
[Docs] Create top level Manage section

### LINKED ISSUE #8718: [Docs] Create top-level Manage section
Move the following files under Manage:
./sources/best-practices/_index.md	
./sources/operations/_index.md	
./sources/operations/authentication.md	
./sources/operations/automatic-stream-sharding.md	
./sources/operations/autoscaling_queriers.md	
./sources/operations/blocking-queries.md	
./sources/operations/loki-canary/_index.md	
./sources/operations/multi-tenancy.md	
./sources/operations/observability.md	
./sources/operations/overrides-exporter.md	
./sources/operations/recording-rules.md	
./sources/operations/request-validation-rate-limits.md	
./sources/operations/scalability.md	
./sources/operations/shuffle-sharding/_index.md	
./sources/operations/storage/_index.md	
./sources/operations/storage/boltdb-shipper.md	
./sources/operations/storage/filesystem.md	
./sources/operations/storage/logs-deletion.md	
./sources/operations/storage/retention.md	
./sources/operations/storage/schema/_index.md	
./sources/operations/storage/table-manager/_index.md	
./sources/operations/storage/tsdb.md	
./sources/operations/storage/wal.md	
./sources/operations/troubleshooting.md	
./sources/upgrading/_index.md

### PR DESCRIPTION
**Pull request by JStickler**
**What this PR does / why we need it**:

Part of the Epic to create a consistent documentation information architecture across the three DB projects.

**Which issue(s) this PR fixes**:
Fixes #8718 

**Special notes for your reviewer**:
The original Issue called for moving files under Manage, but rather than rename the folder and break links, I chose to make the "move" by updating the topic metadata.
This PR also includes updates to topic metadata and removes or updates page weights.

### CODE REVIEW

File: docs/sources/operations/storage/tsdb.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-description: TSDB index
-weight: 1000
+title: Time-series database (TSDB) index store 
+menuTitle:  Time-series database (TSDB) 
+description: Describes the recommended Time Series Database (TSDB) single store index.
+weight: 100
 ---
 
-# TSDB
+# Time-series database (TSDB) index store
```
Comment (chaudum) [inline docs/sources/operations/storage/tsdb.md:8]: See suggestion above

**Review by chaudum — COMMENTED** [review]
General observation: Do we want to follow the pattern that the `description` always starts with "Describes ..."? Most of the pages do that already.

File: docs/sources/operations/authentication.md [hunk 2]
```diff
@@ -1,23 +1,27 @@
 ---
 title: Authentication
-description: Authentication
-weight: 10
+menuTitle:
```
[the remark this answers is not in the document]
Comment (JStickler) [reply docs/sources/operations/authentication.md:3]: Empty menuTitle is not a problem. Hugo uses the title unless there is a value specified for menuTitle, which is essentially an override.

File: docs/sources/operations/storage/boltdb-shipper.md [hunk 3]
```diff
@@ -1,18 +1,24 @@
 ---
-title: Single Store (boltdb-shipper)
-description: Single Store (boltdb-shipper)
+title: Single store (boltdb-shipper)
+menuTitle:  Single store (BoltDB)
+description: Describes the deprecated boltdb-shipper single storage index.
+weight:  200
 ---
-# Single Store (boltdb-shipper)
+# Single store (boltdb-shipper)
```
[the remark this answers is not in the document]
Comment (JStickler) [reply docs/sources/operations/storage/boltdb-shipper.md:7]: Thanks for this!  I went back and forth probably a half dozen times trying to figure out how best to title these two pages.  I like your suggestion.

**Comment by JStickler** [comment]
@chaudum Hugo uses the `Description` in the metadata to show a short, Google style description of the topic on social media (for example, if someone links to a page on Twitter) and it's also used by the search on the docs website. So while it might look weird, having the Description start with "Describes ..."? is actually fine in the output.  What I'm trying to avoid is single word descriptions like in the screen shot below.

description

File: docs/sources/operations/authentication.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (chaudum) [reply docs/sources/operations/authentication.md:3]: Thanks for clarifying!

**Review by chaudum — APPROVED** [review]

**Pull request merged**